src/lib/Liblog/pbs_log.c

```diff
@@ -445,8 +443,7 @@ log_add_debug_info()
 	snprintf(dest, sizeof(dest),
 		"hostname=%s;pbs_leaf_name=%s;pbs_mom_node_name=%s",
 		host, pbs_leaf_name, pbs_mom_node_name);
-	log_record(PBSEVENT_SYSTEM, PBS_EVENTCLASS_SERVER, LOG_INFO,
-		msg_daemonname, dest);
+	log_record_inner(PBSEVENT_SYSTEM, PBS_EVENTCLASS_SERVER, LOG_INFO, msg_daemonname, dest, &mst);
```

Comment:
We need to get the timestamp in log_record() itself to know if we need to switch logs, and then again in log_record_inner() to put in the log record. I was thinking we don't need to call it twice, especially since log_record_inner() is a static method and not an interface.